components/instructions/instructionPanel.tsx

```diff
@@ -43,6 +86,29 @@ export function InstructionPanel() {
                       index={idx + 1}
                       proposalInstruction={pi.info}
                     ></InstructionCard>
+
+                    {pi.info.executionStatus ===
+                      InstructionExecutionStatus.Success &&
+                      notify({
+                        type: 'success',
+                        message: 'Instruction executed successful',
+                        description: 'Instruction executed successful',
+                      })}
+
+                    {showExecuteInstructionModal && (
+                      <Execute
+                        isOpen={showExecuteInstructionModal}
+                        onClose={() => setShowExecuteInstructionModal(false)}
+                        instruction={pi}
+                      />
+                    )}
+
+                    <p
```

Comment:
This button is always visible, please make sure it depends on instruction state (see Oyster)